fix typos in c documentation (#1065)
Codecademy/docs: content/c/concepts/variables/variables.md

---
Title: 'Variables'

> Description: 'A variable is a location in computer memory used to store data, usually for use in a program. To declare a variable in C, the type of variable is first included, followed by the name, and the value. pseudo type name = value;  Though if a variable does not yet need a value assigned to it, this can be omitted and assigned later. c'

Reviewer: ```suggestion
Description: 'A variable is a location in computer memory used to store data.'
```
